frontend/src/components/Header.js

```diff
@@ -2,17 +2,22 @@ import React from 'react';
 import { Flex, Box, Heading, Spacer, ButtonGroup } from '@chakra-ui/react';
 import { Link } from 'react-router-dom';
 
-const Header = () => {
+const Header = ({ isAuthenticated }) => {
 	return (
 		<Flex minWidth="max-content" alignItems="center" bg="blue.300" gap={2} p={2}>
 			<Box p={2}>
 				<Heading size="md" colorScheme="white">Self E-Learning System</Heading>
 			</Box>
 			<Spacer />
 			<ButtonGroup gap={2}>
-				<Link to="/">Dashboard</Link>
-				<Link to="/categories">Categories</Link>
-				<Link to="#">Profile</Link>
+				{ isAuthenticated ? (
+					<React.Fragment>
+						<Link to="/">Dashboard</Link>
+						<Link to="/categories">Categories</Link>
+						<Link to="#">Profile</Link>
+					</React.Fragment>
+					) : null
+				}
```

Comment:
- `!!` is checking if true/false ba si `isAuthenticated` (boolean check)
    - renders the links if true si isAuthenticated otherwise wala
- you can try `<>` and `</>` short-hand version sa `<React.Fragment>` 

```suggestion
				{ !!isAuthenticated && (
					<>
						<Link to="/">Dashboard</Link>
						<Link to="/categories">Categories</Link>
						<Link to="#">Profile</Link>
					</>
					) 
				}
```